draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1752,14 +2109,49 @@ def map2curve25519(r:felem_t) -> felem_t:
     return x
 ~~~
 
+## Finding a curve isogenous to E {#finding-isogeny}
```

Comment:
You're totally right. I added this for completeness, but it may be that this section should go.

The argument in favor of having it is that the description of applying Simplified SWU to pairing-friendly curves is maybe incomplete without something along these lines. But on the other hand, that might be OK---implementors aren't likely to be doing this kind of thing for themselves.

Reply:
There is a section that is missing to be written, which is the instantiations of suites. In such a section, draft must specify all the parameters for a given curve. 
For example, the suite correspondent to BLS12-381 will state which isogeny was used and all other relevant parameters.

Reply:
Great. I made a note of this in #111. Next I'll push a commit that removes this text.

Reply:
(Done)